### ISSUE TITLE
[徵才] 瑞士新創 Upstream Cloud AG 誠徵 Frontend Developer / Full-Stack Developer (full-time/part-time/remote)

### ISSUE THREAD

**Issue by codeewander** [f2etw/jobs#755]
## 團隊簡介/地點
Upstream Cloud AG is the leading audio and video streaming solutions provider in Switzerland. we provide solutions and products for all applications related to streaming of audio and video content. From multi-channel ingest, to transcoding and transmuxing, to worldwide on-demand and live distribution, Upstream Cloud provides everything you need out of one hand.

## 工作內容
**Full Stack Developer**
-  Live your passion for software development building our cloud manager and exciting customer projects and be inspired by your colleagues’ enthusiasm
- Be a hands-on coder applying the best industry standards for code health, security, scalability, and robustness
- Engage in decisions ranging from UX design to back-end architecture together with the tech lead and the team
- Work with agile processes that are adapted to the team’s and project’s needs
- Design and implement new features for our cloud manager, end-to-end, in an iterative and incremental design/build/test cycle
- Build and integrate internal & external (RESTful) APIs
- Lead and execute the realization of customer projects related to video and audio streaming and our cloud manager
- Lend a hand to build one-off landing pages for customer’s livestreams
- Support our customers if they need your help

**Front-end Developer**
- Live your passion for software development building our cloud manager and exciting customer projects and be inspired by your colleagues’ enthusiasm
- Be a hands-on coder applying the best industry standards for code health, security, scalability, and robustness
- Engage in decisions ranging from UX design to API design together with the tech lead and the team
- Work with agile processes that are adapted to the team’s and project’s needs
- Design and implement new features for our cloud manager together with our back-end developers, in an iterative and incremental design/build/test cycle
- Help the other team members with the realization of customer projects related to video and audio streaming and our cloud manager
- Lend a hand to build one-off landing pages for customer’s livestreams
- Support our customers if they need your help

## 能力需求
**Full Stack Developer**
- Strong English and/or German language communication skills
- Passion about writing well-structured, efficient and maintainable code, actively keeping the quality of the code base in check
- Experience with modern front-end SPA development, experience with TypeScript and Angular is a plus
- Experience with API design and good technical understanding of RESTful APIs, preferable with Project Jersey RESTful Web Services in Java
- Experience with SQL and NoSQL databases such as MongoDB and PostgreSQL
- Solid understanding of testing, continuous integration and deployment
- Enjoyment for working in a highly collaborative, fast-paced, and dynamic environment

**Front-end Developer**
- Strong English and/or German language communication skills
- Passion about writing well-structured, efficient and maintainable code, actively keeping the quality of the code base in check
- Experience with modern front-end SPA development and related web development technologies is a must, experience with TypeScript and Angular is a big plus
- A good understanding about consuming RESTful APIs is a must, experience in writing server side scripts with technologies such as PHP is a plus
- Knowledge and interest in practical UI/UX design is a big plus
- Solid understanding of testing, continuous integration and deployment is a plus
- Enjoyment for working in a highly collaborative, fast-paced, and dynamic environment

## 薪資範圍
NT60,000- 120,000. Upon agreement, depending on skills, experience, motivation and drive, creativity,
proficiency in our technology stack, pluses listed, etc.

## 工時/特休
- Full-time (42 hours a week) or part-time (min. 21 hours a week)
- Fully remote or in our office in Hochdorf, Switzerland
- 20 days annual leave plus public holidays in Switzerland

## Why us and what you can look forward to
- Personal & professional development: Through a strong ownership from the first day on you will experience a steep learning curve and grow on a professional level.
- Working schedule: For the most time of the week, you are your own boss, work when and where you feel most productive in order to achieve your goals.
- Working culture: Experience a company culture where we work on eye level, decisions are made fast and knowledge sharing is on our daily agenda.

## 聯絡方式
 Please send your application to jobs@upstream-cloud.ch or contact ri@upstream-cloud.ch

**Comment by Gaitz** [comment]
瑞士的公司聽起來很酷。 不過這個級距的薪水應該沒有 sponsor VISA ?

**Comment by indr** [comment]
> 瑞士的公司聽起來很酷。 不過這個級距的薪水應該沒有 sponsor VISA ?

Thank you for your interest and comment. The position is either remote or in our office in Switzerland. We could definitely help you with the visa application, if you would like to live and work in Switzerland.

**Comment by Gaitz** [comment]
> > 瑞士的公司聽起來很酷。 不過這個級距的薪水應該沒有 sponsor VISA ?
> 
> Thank you for your interest and comment. The position is either remote or in our office in Switzerland. We could definitely help you with the visa application, if you would like to live and work in Switzerland.

This could be a great benefit for some people looking for an overseas job.
Wish you find the right person.

**Comment by indr** [comment]
> I'm looking for a job, is this position still available?

Yes, this position is still available. Thank you for your interest. I'm happy to receive your application at jobs@upstream-cloud.ch. Thank you!
